README.md

```diff
@@ -301,6 +301,20 @@ function look_at_foo(anonymous x: Foo) -> i32 {
         }
     }
 }
+
+enum AlertDescription: i8 {
+    CloseNotify = 0
+    UnexpectedMessage = 10
+    BadRecordMAC = 20
+    // etc
+}
+
+// Use in match:
+function do_nothing_in_particular() => match AlertDescription::CloseNotify {
+   CloseNotify => { ... }
+   UnexpectedMessage => { ... }
+   BadRecordMAC => { ... }
```

Comment:
4 spaces indentation please, even in the readme :D

Reply:
oops~